### PR TITLE
Update version number to check release protocol

### PR DESCRIPTION
**Pull request by andped10**
No changes to code.  Just a check of the release protocol.

Updated `QtInstallerFramework`

Removed `OnlineRepository`
- When installing the app with the new framework it polls for an update and hangs

Updated dependencies

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ build-backend = "hatchling.build"
 # Note that while the project is called EasyReflectometryApp
 # the application itself is EasyReflectometry. 
 name = "EasyReflectometry"
-version = "0.0.7-beta"
+version = "0.0.8-beta"
 description = "Making reflectometry data analysis and modelling easy."
 authors = [
     {name = "Andrew R. McCluskey", email = "andrew.mccluskey@ess.eu"},
```
Comment (rozyczko) [inline pyproject.toml:14]: you should add your name to the list

File: pyproject.toml [hunk 2]
```diff
@@ -41,8 +41,8 @@ dependencies = [
 # These optional dependencies are for building the
 # application as a part of a Github Action.
 ci = [
-    'pyinstaller==6.3.*',
-    'scipy==1.11.*',
+    'pyinstaller==6.4.*',
+    'scipy==1.12.*',
```
Comment (rozyczko) [inline pyproject.toml:45]: these, especially `pyinstaller`, will have to be well tested - we had some problems upgrading pyinstaller in the past.

File: tools/Scripts/MakeInstaller.py [hunk 3]
```diff
@@ -129,15 +129,15 @@ def installerConfigXml():
...
+                # 'RemoteRepositories': {
+                #     'Repository': [
+                #         {
+                #             'Url': onlineRepositoryUrl(),
+                #             'DisplayName': f'{CONFIG.app_name} {CONFIG.setup_os}_{CONFIG.setup_arch} repository',
+                #             'Enabled': 1,
+                #         }
+                #     ]
+                # },
                 'MaintenanceToolName': maintenance_tool_name,
```
Comment (rozyczko) [inline tools/Scripts/MakeInstaller.py:141]: is this Diffraction only? Maybe just delete this block if not necessary

**Review by rozyczko — APPROVED** [review]
Looks necessary for functionality and benign in scope, apart of making sure the installer behaves as expected.

File: pyproject.toml (hunk 2 above)
Comment (andped10) [reply pyproject.toml:45]: I have added a wiki page with the tests:
https://github.com/easyScience/EasyReflectometryApp/wiki/Testing-in-Hand

File: tools/Scripts/MakeInstaller.py (hunk 3 above)
Comment (andped10) [reply tools/Scripts/MakeInstaller.py:142]: I will leave it for now as a reminder that we need to get to a "common" set of tools for building / packaging EasyApplications

**Pull request merged**